src/gsi/setupt.f90

```diff
@@ -1393,7 +1393,7 @@ subroutine init_vars_
         varname='q2m'
         call gsi_bundlegetpointer(gsi_metguess_bundle(1),trim(varname),rank2,istatus)
         if (istatus==0) then
-            if(allocated(ges_z))then
+            if(allocated(ges_q2m))then
```

Comment:
On line 1396, please replace ges_q2m with ges_q2.  There is no variable, ges_q2m, so the code fails to compile.